lapis2/src/main/kotlin/org/genspectrum/lapis/openApi/DownloadAsFileParameterCustomizer.kt

```diff
@@ -0,0 +1,22 @@
+package org.genspectrum.lapis.openApi
```

Comment:
Sadly I still get the problem with swagger, so I could not test it. There is an issue for it #610